src/constants/formData.js

```diff
@@ -0,0 +1,7 @@
+import { HIGH, MEDIUM, LOW } from "./priority";
```

Comment:
instead of exporting/importing the constants 1 by 1, group them as an object. For e.g Priorities. Then use them like this Priorities.High.